cli/command/service/update.go

```diff
@@ -694,7 +694,7 @@ portLoop:
 			if v, ok := portSet[portConfigToString(&port)]; ok {
 				if v != port {
 					fmt.Println("v", v)
-					return fmt.Errorf("conflicting port mapping between %v:%v/%s and %v:%v/%s", port.PublishedPort, port.TargetPort, port.Protocol, v.PublishedPort, v.TargetPort, v.Protocol)
+					return fmt.Errorf("conflicting port mapping between %v:%v/%s/%s and %v:%v/%s/%s", port.PublishedPort, port.TargetPort, port.Protocol, port.PublishMode, v.PublishedPort, v.TargetPort, v.Protocol, v.PublishMode)
```

Comment:
Wondering if this is the right notation. `127.0.0.1:80/tcp` is a standard notation; `127.0.0.1:80/tcp/ingress` looks a little weird. Maybe it should be `127.0.0.1:80/tcp (ingress)`?